packages/gafl-webapp-service/src/processors/licence-type-display.js

```diff
@@ -8,13 +8,13 @@ export const licenceTypeDisplay = permission => {
   if (concessionHelper.hasJunior(permission)) {
     typesStrArr.push('Junior')
   } else if (concessionHelper.hasSenior(permission)) {
-    typesStrArr.push('Senior')
+    typesStrArr.push('Over 65')
```

Comment:
Would be nice to use the SENIOR_MIN_AGE constant from business rules lib for this.

Reply:
Again - I think its circular

Reply:
Not sure what you mean by circular?  I'm only talking about using the SENIOR_MIN_AGE to drive the content - I appreciate it already drives the logic here.